terraform/gitops/generate-files/templates/vnext/vault-secret.yaml.tpl

```diff
@@ -0,0 +1,48 @@
+### hack, this needs to be removed once vnext services are configured with individual mongo user/password/host/etc
```

Comment:
Do we need to raise this issue somewhere in vnext channels / board?

Reply:
yes, this needs to be raised so that the same method is used as in the vnow charts.